src/cli/commands/check.js

```diff
@@ -40,7 +40,7 @@ export async function run(
   }
 
   // get patterns that are installed when running `yarn install`
-  const [, rawPatterns] = await install.hydrate();
+  const [, rawPatterns] = await install.hydrate(true);
```

Comment:
This happens if the fetch is not a part of the check:
```
yarn check v0.18.0-0
error chokidar#fsevents not installed
error fsevents#nan not installed
error fsevents#node-pre-gyp not installed
error node-pre-gyp#npmlog not installed
error node-pre-gyp#rc not installed
error node-pre-gyp#tar-pack not installed
error rc#deep-extend not installed
error tar-pack#fstream-ignore not installed
error rc#minimist not installed
error tar-pack#once not installed
error tar-pack#uid-number not installed
error Found 11 errors.
```

Because the check does not exclude those package due to incompatibility, it expects them to be there, which it shouldn't.